docs/core/reference/cored.md

```diff
@@ -0,0 +1,95 @@
+# cored Command
+
+Command `cored` provides the Chain Core daemon and API server.
+
+`cored` is included with all official releases of Chain Core
+Developer Edition, and is run by default when starting Mac, Windows and Docker.
+
+If you plan to make changes to the internals of Chain Core, you can find
+build instructions in our [GitHub readme](https://github.com/chain/chain/blob/main/Readme.md#building-from-source);
+if you want to _use_ chain core, please download one of our official releases.
+
+### Build tags
```

Comment:
What is shown at startup or `-v` are the weird inversions of the build tags, so it shows `mockhsm` and not `no_mockhsm`. There's one level of indirection there. My 2c would be to punt on presenting these in the doc, but if you anticipate a ton of confusion, we should reflect what is actually printed out.